### ISSUE TITLE
Possibility of setting mouse buttons to specific action buttons when MH is enabled?

### ISSUE THREAD

**Issue by sampconrad-obj** [MouselookHandler/MouselookHandler#15]
Basically title.

Is there a way to set the mouse buttons to alternatively press say, action bar button 1 and 2 when mouse look is enabled?

**Comment by Lagmann** [comment]
> Basically title.
> 
> Is there a way to set the mouse buttons to alternatively press say, action bar button 1 and 2 when mouse look is enabled?

This used to be possible in Shadowlands, but the settings are broken in 10.0. You could type "/MH" and change the settings to actionbutton1 and actionbutton2 for left and right click. I haven't looked at the code yet, but I might attempt to fix it on my own and see if it works.

It would be cool if those were the default instead of strife.

**Comment by Lagmann** [comment]
I was able to change the default keys but the settings are still broken. It's related to ACE3. I'll check it out.

**Comment by Lagmann** [comment]
Alright. I completely fix it on my end.

**Comment by sampconrad-obj** [comment]
Nice. Planning on making a PR? If not, would you be willing to share what you changed to get it working?

**Comment by Lagmann** [comment]
> Nice. Planning on making a PR? If not, would you be willing to share what you changed to get it working?

Yes, I should be able to make a PR tomorrow. I just got busy with other things and couldn't do it today.
